src/misc/KeyManager.ts

```diff
@@ -220,7 +220,7 @@ class KeyManager {
 	}
 }
 
-export function isKeyPressed(keyCode: number, ...keys: Array<Values<typeof Keys>>): boolean {
+export function isKeyPressed(keyCode: string, ...keys: Array<Values<typeof Keys>>): boolean {
```

Comment:
this could use a type that restricts us to the keys we actually use I think, similar to the translation keys. we should be able to extract it from the `Keys` constant with a bit of type magic, or pre-define it as a union and use it in `Keys` and all function signatures.

Reply:
I'm not actually sure that would be needed, keys could be pressed that we don't use and would be passed through here just as part of how we handle key events. Would we have to check if the key was part of this type before passing it in? It just seems unnecessary to add another check.